### DISCUSSION TITLE
problems using previous version of node-windows

### DISCUSSION THREAD

**Discussion by ejesus80** [Ylianst/MeshCentral#5112] [Q&A] [answered]
I'm having problems using an older version of node-windows, I can't use the version of .net that this version requires, is it possible to upgrade to a version of node-windows that uses a current version of .net?

**Comment by SomeGuru** [comment, marked as answer]
Move post over to issues category and ask for enhancement. Delete this discussion and it will get more visual acuity from the community coders.

-SomeGuru
